pages/dashboard.vue

```diff
@@ -9,8 +9,11 @@ import date from "date-and-time";
 import { useDashboard } from "~~/stores/dashboard";
 import gsap from "gsap";
 import { Power2 } from "gsap";
+import Loading from "~/components/Loading.vue";
 import Vue3ChartJs from "@j-t-mcc/vue3-chartjs";
 
+const fetchingData = ref(false);
```

Comment:
We don't need this. See [pending](https://v3.nuxtjs.org/api/composables/use-fetch#return-values).